src/TwigComponent/src/ComponentTemplateFinder.php

```diff
@@ -12,12 +12,20 @@
 namespace Symfony\UX\TwigComponent;
 
 use Twig\Environment;
+use Twig\Loader\LoaderInterface;
 
 /**
  * @author Matheo Daninos <matheo.daninos@gmail.com>
  */
 final class ComponentTemplateFinder implements ComponentTemplateFinderInterface
 {
+    /**
+     * Avoid bug when the lint:command changes the Environment Loader.
+     *
+     * @internal
+     */
+    private LoaderInterface $loader;
+
```

Comment:
Can we, instead, replace the `$environment` constructor argument with `$loader`? We would just need a BC layer (remove the `Environment` type, then have an if statement that checks the type, and trigger deprecation if it is an `Environmnet`).

Reply:
Is Environment|LoaderInterface wrong ? (real question, i'm not really used to BC break handling :) )

Reply:
I think that will work - I still forget we can do this now :)